target/linux/ath79/dts/qca9531_glinet_gl-x300b.dtsi

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+
+/ {
+	keys {
+		compatible = "gpio-keys";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan2g {
+			label = "green:wlan2g";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		wan {
+			label = "green:wan";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+
+		lte {
+			label = "green:lte";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-export {
+		compatible = "gpio-export";
+
+		gpio_minipcie {
+			gpio-export,name = "minipcie";
+			gpio-export,output = <0>;
+			gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
+		};
+
+		gpio_rs485tx_en {
+			gpio-export,name = "rs485tx_en";
+			gpio-export,output = <0>;
+			gpios = <&gpio 1 GPIO_ACTIVE_HIGH>;
+		};
+
+		gpio_ble_rst {
+			gpio-export,name = "ble_rst";
+			gpio-export,output = <0>;
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	watchdog {
+		compatible = "hw_wdt";
+		dog_en_gpio= <12>;
+		feed_dog_gpio=<2>;
+		feed_dog_interval=<100000000>;
+	};
+};
+
+&uart {
+	status = "okay";
+	rs485_pin=<1>;
+};
```

Comment:
status = okay is already present in parent DTS.
I don't know what the other option is good for. If you need it, please fix spaces around = ...

Reply:
I don't know if the setting is necessary, I don't use the RS485 interface and can't test it so I will leave it in, cleanly formatted.

Reply:
Well, if you cannot resolve it, then please don't hide it by marking it as resolved.

Reply:
It wasn't my intention to hide it; it's resolved to the best of my abilities, how should we proceed from here, should it be left in but untested, or should I remove the rs485 pin configuration?

Reply:
Well, it's not resolved. And if we close it, nobody will see the open issue and have the chance to respond and enlighten us.
So, leaving things that are not resolved open seems a good strategy to getting additional input to me ...

BTW, `grep -rn "rs..._pin" target/linux/` does not return anything ...

Reply:
Ok, this is [patch](https://github.com/gl-inet/openwrt/blob/openwrt-19.07.7/target/linux/ath79/patches-4.14/821-fix-glinet-rs485-auto-txrx.patch#L75-L78) is present in the GL.iNet source and makes use of the `rs485_pin` setting.

I don't intend to bring this patch over; I'll remove the setting. I'm not certain if the uart is still needed; other dts files for devices with functional serial ports don't include a uart section, I believe because of your [commit](https://github.com/openwrt/openwrt/commit/3a4b751110b32380cf827f14a203190cfbc2e40f) to centralize it in the `ar9330.dtsi` file.

Please confirm if my understanding is correct and I'll remove the entire section.

Thanks for again for your time.

Reply:
I've removed the uart section and the rs485 pin, I've also updated the commit message to reflect the untested nature of the RS485 support and possible need for the patch referenced above.